README.md

```diff
@@ -75,5 +75,7 @@ Tokite support only below events now.
 <tr><td>user:</td><td>Match user name.</td><td>user:hogelog</td></tr>
 <tr><td>label:</td><td>Match pull_request or issue label.</td><td>label:Feature</td></tr>
 <tr><td>review_state:</td><td>Match pull_request_review state.</td><td>review_state:/commented|approved|changes_requested/</td></tr>
+<tr><td>requested_reviewer:</td><td>Match user name of review requested reviewer</td><td>requested_reviewer:hogelog</td></tr>
+<tr><td>requested_team:</td><td>Match team slug of review requested team</td><td>requested_team:chef</td></tr>
```

Comment:
Q. How to handle same name teams in different orgs? For instance, we can have `@org-a/dev` and `@org-b/dev` at the same time, but their `slug` is the same, `dev`. Should we use `repo` matcher together with `requested_team` matcher?

Reply:
Now we can use full team name like `org-a/dev` to match requested team.
https://github.com/cookpad/tokite/pull/53/commits/f3ced22b589bd3704088ce5ecef6e7070258f98e